EMML/1001-2000/EMML1939.xml

```diff
@@ -35,51 +35,268 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                   <collection>Parish Church, Ambāssal</collection>
                   <idno>MS EMML no. 1939</idno>
                </msIdentifier>
-               <msContents>
-                  <summary/>
-               </msContents>
+
+<physDesc>
+                 <objectDesc form="Codex">
+                    <supportDesc>
+                       <support>
+                          <material key="parchment"/>
+                       </support>
+                       <extent>
+                          <measure unit="leaf">174</measure>
+                          <dimensions type="outer" unit="cm">
+                             <height>28</height>
+                             <width>20</width>
+                          </dimensions>
+                       </extent>
+                       <foliation/>
+                       <condition key="good"/>
```

Comment:
This is of course your decision, but I think the condition of the ms could be described in more details. I have seen some damaged leaves, stains and repairs. But this is of course up to you